app/views/form/check_answers.html.erb

```diff
@@ -17,10 +17,16 @@
     <% end %>
     <%= display_answered_questions_summary(subsection, @case_log, current_user) %>
 
-    <%= render partial: "form/check_answers_summary_list", locals: {
-      subsection:,
-      case_log: @case_log,
-    } %>
+    <% if subsection.applicable_questions(@case_log).map(&:check_answers_card_number).compact.length > 0 %>
```

Comment:
Might be worth extracting a helper method here?

Reply:
Agreed, will do that now 🙂